Cython/Compiler/Tests/TestGrammar.py

```diff
@@ -174,6 +181,13 @@ def test_valid_number_literals(self):
     def test_invalid_ellipsis(self):
         ERR = ":{0}:{1}: Expected an identifier or literal"
         for code, line, col in INVALID_ELLIPSIS:
+            try:
+                ast.parse(textwrap.dedent(code))
+            except SyntaxError as exc:
+                assert "invalid syntax" in str(exc), str(exc)
```

Comment:
I'd rather not check for specifics of the error message here. There have been huge changes in CPython since the parser rewrite, and improving error messages is a major goal for them, especially regarding syntax errors and suggestions about what might be wrong. There's quite a chance that this message might change in the future.
```suggestion
                assert True
```